R/deps.R

```diff
@@ -4,7 +4,7 @@
 #' @param package The package (or packages) to search for reverse dependencies.
 #' @inheritParams revdep_check
 #' @export
-cran_revdeps <- function(package, dependencies = TRUE, bioc = FALSE) {
+cran_revdeps <- function(package, dependencies = TRUE, bioc = TRUE) {
```

Comment:
I don't think this is needed. This is an exported function, so strictly speaking it would also be breaking change.